File: app/src/main/java/eu/warble/pjapp/util/Tools.java
package eu.warble.pjapp.util;


import eu.warble.pjapp.data.model.Student;

public class Tools {

    public static boolean checkApiResponseForErrors(Student student) {
        return !(student.getImie() == null || student.getNazwisko() == null) &&
                !(student.getStudia() == null || student.getZajecia() == null);
    }

}

File: app/src/main/java/eu/warble/pjappkotlin/utils/Tools.kt
package eu.warble.pjappkotlin.utils

import eu.warble.pjappkotlin.data.model.Student
import android.text.Html
import android.os.Build
import android.text.Spanned

object Tools {
    fun checkApiResponseForErrors(student: Student): Boolean {
        return !(student.imie == null || student.nazwisko == null) &&
                !(student.studia == null || student.zajecia == null)
    }

    @Suppress("DEPRECATION")
    fun fromHtml(source: String): Spanned {
        return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
            Html.fromHtml(source, Html.FROM_HTML_MODE_LEGACY)
        } else {
            Html.fromHtml(source)
        }
    }
}